perception/elevation_map_loader/src/elevation_map_loader_node.cpp

```diff
@@ -128,6 +151,21 @@ void ElevationMapLoaderNode::publish()
     pcl::toROSMsg(*elevation_map_cloud_ptr, elevation_map_cloud_msg);
     pub_elevation_map_cloud_->publish(elevation_map_cloud_msg);
   }
+  is_elevation_map_published_ = true;
+}
+
+void ElevationMapLoaderNode::timerCallback()
+{
+  // flag to make receiveMap() called only once.
+  static bool is_map_received = false;
```

Comment:
I think better to put it as a member variable.

Reply:
Apply it in https://github.com/autowarefoundation/autoware.universe/pull/3162/commits/5ee3685f8f63f3b0183f4dda49f447e971cc39d6